agent/pom.xml

```diff
@@ -59,28 +58,41 @@
 	</organization>
 	<issueManagement>
 		<system>JIRA</system>
-		<url>https://bugs.openjdk.java.net/projects/JMC/issues</url>
+		<url>https://bugs.openjdk.org/projects/JMC/issues</url>
 	</issueManagement>
 	<mailingLists>
 		<mailingList>
 			<name>jmc dev</name>
-			<subscribe>http://mail.openjdk.java.net/mailman/listinfo/jmc-dev</subscribe>
-			<unsubscribe>http://mail.openjdk.java.net/mailman/listinfo/jmc-dev</unsubscribe>
-			<archive>http://mail.openjdk.java.net/pipermail/jmc-dev/</archive>
+			<subscribe>https://mail.openjdk.org/mailman/listinfo/jmc-dev</subscribe>
+			<unsubscribe>https://mail.openjdk.org/mailman/listinfo/jmc-dev</unsubscribe>
+			<archive>https://mail.openjdk.org/pipermail/jmc-dev/</archive>
 		</mailingList>
 	</mailingLists>
 	<properties>
+		<!-- Config -->
 		<revision>1.0.1</revision>
 		<changelist>-SNAPSHOT</changelist>
+		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+		<project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
 		<maven.compiler.source>11</maven.compiler.source>
 		<maven.compiler.target>11</maven.compiler.target>
+		<jmc.config.path>${project.basedir}/../configuration</jmc.config.path>
+		<!-- Plugin Versions -->
+		<maven.checkstyle.version>3.3.0</maven.checkstyle.version>
```

Comment:
Overall the changes look good to me, but while you're here changing the pom, what are your thoughts on alphabetical ordering the plugins?

Reply:
Well, alphabetical ordering could be an idea. Or maybe ordering in the order that plugins appear in the build - although this is difficult since certain plugins might appear in different phases. So, maybe alphabetical ordering would be good. But I would defer this to another PR.

Reply:
I think ordering in order of how they appear in the build would be pretty nice too, although that might be difficult to maintain over time and adds an extra thing to keep in the back of the mind when adding new plugins. Alphabetical just came to mind because there have been a couple (many) of times where I've gone sifting through the poms to check for version numbers and thought it might be nice to have a consistent and organized way to do so.

Reply:
OK, I keep this in mind and will maybe come up with a proposal that is consistent through all of the poms.